src/js/apis/c2c/TagsService.js

```diff
@@ -0,0 +1,30 @@
+function TagsService(api) {
+  this.api = api;
+}
+
+TagsService.prototype.add = function (document) {
+  if (document.type !== 'r') {
+    // As for now, only routes may be tagged.
+    return Promise.reject(new Error('Tags are not supported for this type of documents'));
+  }
+
+  const data = {
+    document_id: document.document_id,
+  };
+
+  return this.api.post('/tags/add', data);
+};
+
+TagsService.prototype.remove = function (document) {
+  const data = {
+    document_id: document.document_id,
+  };
```

Comment:
Here also, I would use destructuring (althoug the argument type is then less clear because we use no jsdoc or typescript), or at least there is no need to instantiate `data` variable, just put this directly as argument for `post` method